package.json

```diff
@@ -21,10 +21,12 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-news#readme",
   "devDependencies": {
+    "express": "^4.18.2",
     "husky": "^8.0.2",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
-    "pg-format": "^1.0.4"
+    "pg-format": "^1.0.4",
+    "supertest": "^6.3.4"
```

Comment:
which of these dependencies will actually be needed in running the server in a production environment (and therefore shouldn't be dev dependencies)?